lib/trailblazer/generator/generate.rb

```diff
@@ -0,0 +1,48 @@
+module Trailblazer
+  # Trailblazer Gen
+  class Generator
+    module Generate
+      extend Trailblazer::Activity::Railway()
+
+      REGEXP_CONCEPT = /^[A-Z][A-Za-z]{1,}(::[A-Z][A-Za-z]{1,})?/
+      REGEXP_NAME = /^[A-Z][A-Za-z]*$/
+
+      module_function
+
+      def validate_concept(_ctx, concept:, **)
+        concept.match REGEXP_CONCEPT
+      end
+
+      def validate_class_name(_ctx, options:, type:, **)
+        return Trailblazer::Activity::Right if type == :concept
+        options[:name].match REGEXP_NAME
+      end
+
+      def init(ctx, options:, concept:, type:, **)
+        concept = Utils::Fetch.concept(concept) unless type == :concept
+        ctx[:context] = Utils::Fetch.context(options, type, concept)
+      end
+
+      def generate_concept?(_ctx, type:, **)
+        type == :concept
+      end
+
+      def generate_concept(ctx, context:, **)
+        ctx[:destination] = destination = Concept.dir(context.concept_path)
+        Utils::Files.mkdir(destination)
+      end
+
+      # NOTE: why do I have to specifcy Trailblazer::Activity::End here and not in GenerateFile activity??
+      step method(:validate_concept), Output(:failure) => Trailblazer::Activity::End(:wrong_concept_format)
+      step method(:validate_class_name), Output(:failure) => Trailblazer::Activity::End(:wrong_class_name_format)
+      pass method(:init)
+      step method(:generate_concept?), Output(:success) => :generate_concept_step, Output(:failure) => :generate_file
+      step task: GenerateFile, id: :generate_file,
```

Comment:
Note sure about this...I want to manually wiring the ends in order to expose the subprocess ends to the main activity but following the docs (http://trailblazer.to/api-docs/#activity-composition-example-subprocess) and using `Track(:seccess)` seems not doing what *I think* should do. When I inspect the circuit I can see that using `Track` will create a loop with itself the right track of this step will be connected to `GenerateFile` again....
What am I missing here?